### PR TITLE
Add documentation for fritz lights

### PR DESCRIPTION
**Pull request by flabbamann**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/52830
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/fritzbox.markdown [hunk 1]
```diff
@@ -82,3 +85,21 @@ There are several attributes that can be useful for automations and templates.
 | `locked` | The state of the lock for configuring the device via the app or the FRITZ!Box web interface.
 | `temperature_unit` |  The unit of the temperature sensor.
 | `temperature` | The current temperature sensor reading.
+
+## Light
+
+To get AVM FRITZ!DECT lightbulbs (e.g.,  FRITZ!DECT 500) follow the [configuration instructions](#configuration) above.
```
Comment (frenck) [inline source/_integrations/fritzbox.markdown]:
```suggestion
To get AVM FRITZ!DECT lightbulbs (e.g., FRITZ!DECT 500) follow the [configuration instructions](#configuration) above.
```

File: source/_integrations/fritzbox.markdown [hunk 2]
```diff
@@ -82,3 +85,21 @@ There are several attributes that can be useful for automations and templates.
...
+
+### Attributes
+
+There are several attributes that can be useful for automations and templates. See [`Light`](/integrations/light/) integration for details.
+
+| Attribute | Description |
+| --------- | ----------- |
+| `hs_color` | The hue and saturation values for the color. See note above.
+| `color_temp` | The lights color temperature
+| `brightness` |  The lights brightness
```
Comment (frenck) [inline source/_integrations/fritzbox.markdown]: These are not specific to this integration.

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/fritzbox.markdown (hunk 2 above)
Comment (flabbamann) [reply source/_integrations/fritzbox.markdown]: I wasn't sure here. Thanks 👍

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — APPROVED** [review]

**Pull request merged**